desktop/components/split_test/running_tests.coffee

```diff
@@ -14,4 +14,9 @@
 # this should export empty Object
 # module.exports = {}
 
-module.exports = {}
+module.exports =
```

Comment:
Yep, that way we can iterate over multiple PRs (and not have to get everything ready to go all in one).